pkgs/by-name/tr/tracexec/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
```

```suggestion
{
  lib,
  stdenv,
  fetchFromGitHub,
  rustPlatform,
}:
```

Formatted as per RFC 0166